libusb/libusb.h

```diff
@@ -535,7 +536,10 @@ enum libusb_bos_type {
 	LIBUSB_BT_CONTAINER_ID = 0x04,
 
 	/** Platform descriptor */
-	LIBUSB_BT_PLATFORM_DESCRIPTOR = 0x05
+	LIBUSB_BT_PLATFORM_DESCRIPTOR = 0x05,
+	
+	/* SuperSpeed+ device capability */
+	LIBUSB_BT_SUPERSPEED_PLUS_CAPABILITY = 0x0A
```

Comment:
nit: I suggest a trailing comma on the last entry. It means future additions won't have to touch the 2nd to last item, which makes diffs larger and pollutes blame history.